src/filedef.cpp

```diff
@@ -1526,6 +1526,18 @@ void FileDefImpl::acquireFileVersion()
   if (!vercmd.isEmpty() && !m_filePath.isEmpty() &&
       m_filePath!="generated" && m_filePath!="graph_legend")
   {
+    const StringVector &tagFileList = Config_getList(TAGFILES);
```

Comment:
I'd expect this loop can be simplified by just checking for `isReference()`, or do you see a case where that would not work?

Reply:
I'm not sure whether or not the the reference (i.e. just above it by means of `setReference(lref)`) is set or not. This should be checked by means of the example.

A quick test with the example shows that the `p1.h` in is set as reference, so should be OK.